app/serializers/user_serializer.rb

```diff
@@ -1,8 +1,10 @@
 class UserSerializer < ActiveModel::Serializer
   attributes :id, :cover_image_url, :avatar_template, :rating_type, :mini_bio,
     :about, :is_followed, :title_language_preference, :location, :website,
-    :waifu, :waifu_or_husbando, :waifu_slug, :waifu_char_id, :last_sign_in_at,
-    :current_sign_in_at, :is_admin, :following_count, :follower_count, :is_pro
+    :waifu_or_husbando, :last_sign_in_at, :current_sign_in_at, :is_admin,
+    :following_count, :follower_count, :is_pro
+
+  has_one :waifu, root: :characters
```

Comment:
This will cause an N+1 on pages where we load in multiple users, like the feeds.  Waifu should probably be moved to UserInfo.  Wasn't there something we were discussing about refactoring that? `User` and `FullUser` maybe?